.semaphore/semaphore.yml

```diff
@@ -17,7 +17,7 @@ global_job_config:
       - sem-version node 12.16.1
       - export BRANCH_TAG=${SEMAPHORE_GIT_BRANCH/\//-}
       - export CACHE_NAME=$SEMAPHORE_GIT_BRANCH-$SEMAPHORE_PIPELINE_ID
-      - echo "$DOCKER_HUB_TOKEN" | docker login --username=$DOCKER_HUB_USERNAME --password-stdin
+      - echo "$DOCKER_TOKEN" | docker login $DOCKER_REGISTRY --username=$DOCKER_USERNAME --password-stdin
```

Comment:
How about?
```suggestion
      - echo "$DOCKER_REGISTRY_TOKEN" | docker login $DOCKER_REGISTRY --username=DOCKER_REGISTRY_USERNAME --password-stdin
```

Reply:
Renamed in https://github.com/nimblehq/rails-templates/pull/224/commits/a69ce0df6b0519298835c98ac0650ec779d12cfd